docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: "3.9"
+services:
+  pg:
+    image: postgres:15.2
```

Comment:
The image gets pulled from Docker Hub, see [here](https://hub.docker.com/_/postgres)